Point readers to a more detailed source of info

waldoj opened the pull request:
This is not the most elegant way to direct people to the travel guide, but this _is_ the page's introductory text, so I can't find a better place to put it.

afeld commented:
I didn't check that all of the information in this section is reflected in the new one, but taking your word for it!

waldoj committed:
Point readers to a more detailed source of info

This is not the most elegant way to direct people to the travel guide, but this _is_ the page's introductory text, so I can't find a better place to put it.
